### PR TITLE
javascript-and-json

### PR DESCRIPTION
**Pull request by easton-morris**
image

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+var arrayString = JSON.stringify(array);
+
+console.log('JSON String:', arrayString, typeof arrayString);
+
+var student = '{"student":[{"id": 456,"name": "James Halliday"}]}';
+
+console.log('Future Object in JSON', student, typeof student);
+
+var JSONobject = JSON.parse(student);
```
Comment (BrettAlbright) [inline javascript-and-json/main.js]: The only variables which should be capitalized should be ES6 classes. Normal vars should be lowercase.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
tiny update!

File: javascript-and-json/main.js (hunk 1 above)
Comment (easton-morris) [reply javascript-and-json/main.js]: true! haha. mb i missed that.  fixed!

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**